keyboards/nyquist/rules.mk

```diff
@@ -41,9 +41,13 @@ F_USB = $(F_CPU)
 
 # Bootloader
 #     This definition is optional, and if your keyboard supports multiple bootloaders of
-#     different sizes, comment this out, and the correct address will be loaded 
+#     different sizes, comment this out, and the correct address will be loaded
 #     automatically (+60). See bootloader.mk for all options.
-BOOTLOADER = caterina
+ifeq ($(strip $(KEYBOARD)), nyquist/rev3)
+    BOOTLOADER = dfu
```

Comment:
yeessss, a decent bootloader. 

Can someone actually still put a pro micro in though?

Reply:
Nyquist 3 actually has an on-board controller, so someone would have to go through a considerable effort to put a Pro Micro on there 😄